com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -317,10 +320,14 @@ private static NetworkEndpoint ParseNetworkEndpoint(string ip, ushort port, bool
                 if (!NetworkEndpoint.TryParse(ip, port, out endpoint, NetworkFamily.Ipv4) &&
                     !NetworkEndpoint.TryParse(ip, port, out endpoint, NetworkFamily.Ipv6))
                 {
+#if HOSTNAME_RESOLUTION_AVAILABLE && UTP_TRANSPORT_2_4_ABOVE
+                    return default;
+#else // If the user does not have the most recent version of UnityTransport installed
                     if (!silent)
```

Comment:
Should the `return default` be after the `if (!silent)` check?

Reply:
Applied the same adjustments made to v2.x branch for this.